miden-crypto/src/word.rs

```diff
@@ -149,7 +154,7 @@ impl Randomizable for RpoDigest {
 
 /// Errors that can occur when working with RpoDigest.
```

Comment:
```suggestion
/// Errors that can occur when working with a [Word].
```
nit: a search of the repository shows that there are a few other uses of `RpoDigest` in comments elsewhere in the codebase